spec/controllers/corporate_controller_spec.rb

```diff
@@ -1,4 +1,4 @@
-RSpec.describe CorporateController, type: :controller do
+RSpec.describe CorporateController, type: :controller, features: [:corporate] do
```

Comment:
There are a couple of other files which follow the same format. I personally don't mind whether we use arrays or not (for single features) but I'd like consistency. I suppose having single features in an array clarify more intently that multiple features can be passed in. What do you reckon?